atomics/T1012/T1012.yaml

```diff
@@ -126,3 +126,11 @@ atomic_tests:
       reg.exe query hklm\software\microsoft\windows\softwareinventorylogging /v collectionstate /reg:64
     name: command_prompt
     elevation_required: true
+- name: Inspect SystemStartOptions Value in Registry
+  description: The objective of this test is to query the SystemStartOptions key under HKLM\SYSTEM\CurrentControlSet\Control in the Windows registry. This action could be used to uncover specific details about how the system is configured to start, potentially aiding in understanding boot parameters or identifying security-related settings.
+    key is.
+  supported_platforms:
+  - windows
+  executor:
+    name: command_prompt
+    command: reg.exe query HKLM\SYSTEM\CurrentControlSet\Control /v SystemStartOptions
```

Comment:
Hello @krdmnbrk : the keys here are a bit mis aligned. It should have a structure something like this :
```
  executor:
    command: |
     reg.exe query HKLM\SYSTEM\CurrentControlSet\Control /v SystemStartOptions
    name: command_prompt
    elevation_required: true
```

but

Reply:
Hey @patel-bhavin, thank you for the correction. Actually, I thought there was no need to put “|” as it is not mandatory when there is not more than one line. The related atomic works this way, but the part you mentioned is also in the [spec file](https://github.com/redcanaryco/atomic-red-team/wiki/Sample-Spec). I missed it. I fixed it.